### ISSUE TITLE
branch: tests

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/babel#2]
Looks like this was created by @aaronsamuel137.  Please describe this branch and its plans for the future.

**Comment by aaronsamuel137** [comment]
This branch was created for testing committing to babel from rosetta directly using the gtihub API. It is not intended to ever be merged into master.

**Comment by aaronsamuel137** [comment]
Leaving this issue open but unassigning since there is nothing to do here but document the branch.

**Comment by jbphet** [comment]
@aaronsamuel137 - can this branch be deleted now that Rosetta has been deployed, or do we need to keep it around long term as a record of the testing that has been done?

**Comment by aaronsamuel137** [comment]
I don't think we need a record of testing, but I could see it continuing to be useful for testing as we add new features.

Currently, rosetta running on simian is set to use branch tests so any new features can be easily tested there without messing up the strings in master.

**Comment by jbphet** [comment]
Okay, sounds like this branch will be long lived and will serve a role in future testing.
